lamindb_setup/_connect_instance.py

```diff
@@ -219,12 +220,20 @@ def connect(instance: str | None = None, **kwargs) -> str | tuple | None:
     _user: UserSettings | None = kwargs.get("_user", None)
     if _user is not None:
         access_token = _user.access_token
+    if (
+        settings.user.api_key is None
+        and settings.user.password is None
+        and os.environ.get("LAMIN_API_KEY") is not None
+    ):
+        login()
+    if instance is None:
+        instance = os.environ.get("LAMIN_CURRENT_INSTANCE")
```

Comment:
```suggestion
        instance = os.environ.get("LAMIN_CURRENT_INSTANCE", None)
```
Just for clarity.

Reply:
Hm, I had with `None` before but in fact I prefer it without `None` because it's a very well known convention.